packages/common/src/sites/_internal/applyDefaultCollectionMigration.ts

```diff
@@ -4,6 +4,17 @@ import { WellKnownCollection } from "../../search/wellKnownCatalog";
 import { IModel } from "../../types";
 import { SearchCategories } from "./types";
 
+export const defaultSiteCollectionKeys: WellKnownCollection[] = [
+  // TODO: add 'all' as a wellknown collection and figure out the
+  // ramifications of doing so across the app. (or create a new
+  // type that includes 'all')
+  "all" as any,
+  "dataset",
+  "document",
+  "site",
+  "appAndMap",
+];
```

Comment:
Totally good for now, but given the dev sync today should we move this out into it's own file?